src/api/app/controllers/webui/project_controller.rb

```diff
@@ -654,7 +654,9 @@ def load_project_info
     @linking_projects = @project.linked_by_projects.pluck(:name)
 
     reqs = @project.open_requests
-    @requests = (reqs[:reviews] + reqs[:targets] + reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
+    @incoming_requests = reqs[:incidents] | reqs[:maintenance_release]
+    @outgoing_requests = reqs[:targets] | reqs[:reviews]
+    @requests = @incoming_requests + @outgoing_requests
```

Comment:
as @Ana06 said, would be better to have:

```
@requests = @incoming_requests | @outgoing_requests 
```
in this way you are selecting just the unique requests

Reply:
Sure, but still, it's the core logic is broken. `@incoming_requests` doesn't contain incoming requests and `@outgoing_requests` doesn't contain outgoing requests. They're all mixed up. See [this comment](https://github.com/openSUSE/open-build-service/pull/7516#discussion_r283229697)